Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.yml

```diff
@@ -0,0 +1,125 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: JSONFeed
+  version: -1
+configuration:
+- display: Fetch indicators
+  name: feed
+  defaultvalue: true
+  type: 8
+  required: false
+- display: Indicator Reputation
+  name: feedReputation
+  defaultvalue: feedInstanceReputationNotSet
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  additionalinfo: Indicators from this integration instance will be marked with this reputation
+- defaultvalue: F - Reliability cannot be judged
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+  additionalinfo: Reliability of the source providing the intelligence data
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: suddenDeath
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- name: feedExpirationInterval
+  defaultvalue: "20160"
+  required: false
+  type: 1
+  display: ""
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- display: Bypass exclusion list
+  name: feedBypassExclusionList
+  defaultvalue: ""
+  type: 8
+  required: false
+  additionalinfo:
+    When selected, the exclusion list is ignored for indicators from this feed.
+    This means that if an indicator from this feed is on the exclusion list,
+    the indicator might still be added to the system.
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: URL
+  name: url
+  required: true
+  type: 0
+- additionalinfo: Type of the indicator in the feed.
+  display: Indicator Type
+  name: indicator_type
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: false
+  type: 9
+- additionalinfo: JMESPath expression for extracting the indicators from.
```

Comment:
```suggestion
- additionalinfo: JMESPath expression for extracting the indicators.
```

Reply:
Please include additional information here in the tooltip, e.g. a link to JMESPath expression syntax perhaps and/or an example.